src/index.css

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+
+* {
+  background-color: #ffd6d6;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.imagem {
+  width: 256px;
+  height: 144px;
+  justify-content: flex-start;
+  margin-right: 10px;
+}
+
+.icon {
+  width: 20px;
+  height: 20px;
+  background-color: inherit;
+}
+
+.heroes-grid-container {
+  display: flex;
+  justify-content: center;
+}
+
+#grid {
+  width: 1300px;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.hero {
+  padding: 0 20px 20px 20px;
+}
+
+.heroInfo {
+  text-align: center;
+}
+
+.hero-name {
```

Comment:
There is a number of ways to add styles.
In our actual codebase, in my opinion, the styles are better organized when they are close to the component that uses this styles, what do you think?
It would be like this:
```
src/
  - Header.js
  - Header.css
```
and inside `Header.css` you are focused only in 1 component.

Adding a lot of styles in one big file would become hard to maintain, easily this file would be 100000 lines long